django/db/api.py

```diff
@@ -134,8 +135,17 @@ class Meta:
         
     def dehydrate(self, bundle):
         bundle.data = get_detail_bundle(bundle.obj, ['cell',''])
+        _filter = ( 
```

Comment:
You should be able to use a nested function here for a bit more clarity, i.e. "def _filter(field_information): ..." but this is fine.